src/components/SiteInfoInputForm.js

```diff
@@ -760,8 +759,14 @@ const styles = {
         justifyContent: 'center',
         alignSelf: 'center',
     },
-    largeTextInput: {
-        height: 100
+    textInputStyle: {
+        minHeight: 40,
+        fontSize: 18,
+        marginLeft: 10,
+        borderColor: '#86939e',
+        borderBottomWidth: 1,
+        width: '90%',
+        paddingBottom: 5
```

Comment:
Manual styling to get the `TextInput` to look like the `react-native-elements` `Input`